esp32/modules/easyrtc.py

```diff
@@ -18,15 +21,15 @@ def string(date=False, time=True):
     hourstr = str(hour)
     if (hour<10):
       hourstr = "0"+hourstr
-    minstr = str(min)
-    if (min<10):
+    minstr = str(minute)
```

Comment:
It's three bytes longer. I don't see why you would even write a whole comment to address this nonexistent problem.
I now call it minute. Deal with it...

(•_•) 
( •_•)>⌐■-■ 
(⌐■_■)